hazelcast-sql/src/main/java/com/hazelcast/sql/impl/schema/MappingField.java

```diff
@@ -89,8 +89,10 @@ public MappingField setType(QueryDataType type) {
     }
 
     /**
-     * The external name of a field. For example, in case of IMap or Kafka,
-     * it always starts with `__key` or `this`.
+     * Returns what is specified after {@code EXTERNAL NAME} keyword in the corresponding
```

Comment:
Previous javadoc explanation was clearer, at least to me. Would you consider rephrase it better or even revert the change?

Reply:
The previous explanation wasn't clear about whether the external name is a computed or user-defined value. This version also says how/where it is specified. I had to look at all usages of `externalName()` to understand what it really is, and came up with this explanation.